hazelcast/src/main/java/com/hazelcast/config/HotRestartPersistenceConfig.java

```diff
@@ -90,6 +98,21 @@ public HotRestartPersistenceConfig setBaseDir(File baseDir) {
     }
 
     /**
+     * Gets the configured number of Hot Restart store instance to create for one Hazelcast instance.
+     */
+    public int getParallelism() {
+        return parallelism;
+    }
+
+    /**
+     * Sets the number of Hot Restart store instances to create for one Hazelcast instance.
+     */
+    public HotRestartPersistenceConfig setParallelism(int parallelism) {
```

Comment:
It is validated deep within the startup process, at the point where we have enough information to do a complete validation. 

https://github.com/mtopolnik/hazelcast-enterprise/blob/concurrent-hr-store/hazelcast-enterprise/src/main/java/com/hazelcast/spi/hotrestart/HotRestartService.java#L165